release-notes/release-notes509.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: TinyMCE 5.0.9
+title_nav: TinyMCE 5.0.9
+keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
+---
+
+## Overview
+
+The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.9 and document known problems in this release, as well as important bug fixes, deprecated functionality, and other details.
+
+TinyMCE 5.0.9 release adds improvements to the TinyMCE editor to improve the overall user experience.
+
+> **Note:** TinyMCE 5.0.8 community was not released to enterprise due to blockers found during QA. As such TinyMCE 5.0.9 enterprise contains changes from both TinyMCE 5.0.8 and 5.0.9 community releases.
+
+## New features
+
+### TinyMCE
+
+#### Multiple toolbars
+
+TinyMCE 5.0.9 adds back the ability to include multiple toolbars in the editor, similar to TinyMCE 4.x. Multiple toolbars can be configured by using the toolbar array or toolbar1, toolbar2, etc... configuration options.
+
+![Multiple toolbars]({{site.baseurl}}/images/multiple-toolbars.png)
+
+For more information on multiple toolbars refer to the [documentation]({{site.baseurl}}/configure/editor-appearance/#usingmultipletoolbars).
+
+> **Note:** If multiple toolbars are used, the toolbar drawer will be disabled as toolbar drawers aren't compatible with multiple toolbars.
+
+#### base_url and suffix options
+
+TinyMCE 5.0.9 adds two new configuration options: `base_url` and `suffix`. These new options provide an easier way to provide the editor manager `baseURL` and `suffix` properties.
+
+For more information on the new options refer to the [base_url]({{site.baseurl}}/configure/integration-and-setup/#base_url) and [suffix]({{site.baseurl}}/configure/integration-and-setup/#suffix) documentation.
+
+#### Media plugin m4a file support
+
+The **Media** plugin now supports the `M4A` audio format. Inserting a m4a file will now be embedded using a HTML5 `<audio>` element.
+
+For more information on the media plugin refer to the [documentation]({{site.baseurl}}/plugins/media/).
+
+## Updates and enhancements
+
+### TinyMCE
+
+The new TinyMCE 5.0.9 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the [changelog]({{site.baseurl}}/changelog/#version509june262019).
```

Comment:
...This paragraph is getting rather like a broken record. We could perhaps just use the last sentence with a few words added? The rest isn't super necessary.

Reply:
Yeah, I feel the same way, but it's been there for every version atm so I'm hesitant to remove it tbh. Maybe we can remove the middle sentence, as I think that's the worse part and really doesn't make sense to me?